src/main/java/com/google/api/generator/gapic/composer/comment/ServiceClientCommentComposer.java

```diff
@@ -195,9 +195,9 @@ public static CommentStatement createCreateMethodStubArgComment(
     return comments;
   }
 
- public static List<CommentStatement> createRpcMethodHeaderComment(Method method) {
-    // TODO(summerji): Refactor this method when implement default method sample code.
```

Comment:
Still needed?

Reply:
Yes. The [caller](https://github.com/googleapis/gapic-generator-java/blob/merge_master/src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java#L681) `createMethodDefaultMethod` pass down the sample code of default value into the default method's comment composer.

Reply:
Oh, we merge this change in `samplecode-master` https://github.com/googleapis/gapic-generator-java/pull/594/files#diff-54602d8fcebb00c962bb8da32989461212103a71ce60ac29c49e3fe6cf945e4eR652.
I guess I lost part of PR. I will make it up.